server/views/resources/how-nonprofit-projects-work.jade

```diff
@@ -0,0 +1,47 @@
+extends ../layout
+block content
+    .col-xs-12.col-sm-8.col-sm-offset-2.col-md-6.col-md-offset-3
+        html.
+            <h2 class='text-center'>How our Nonprofit Projects Work</h2>
+            <hr>
```

Comment:
That's not necessary. We should stick with jade unless there is some other reason.